src/main/java/com/coupang/marketplace/product/repository/ProductRepository.java

```diff
@@ -0,0 +1,23 @@
+package com.coupang.marketplace.product.repository;
+
+import com.coupang.marketplace.product.domain.Product;
+import org.apache.ibatis.annotations.Mapper;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+
+@Mapper
+@Repository
+public interface ProductRepository {
+
+    List<Product> getProductsByIsRocket(boolean isRocket, int pageStart, int listSize);
```

Comment:
제가 돌려보니 컴파일러가 파라미터를 발견하지 못해 오류가 발생하네요😊
`@Param("isRocket")` 와 같이 파라미터를 명시해주어야 할 것 같습니다!

Reply:
앗 이건 제가 아까 리팩토링하다가 중간에 끊겨서 그런 것 같습니다!
래퍼클래스로 수정했더니 널값이 들어가는 바람에 NPE 가 발생해서 그렇습니다! 이 부분은 수정해서 올렸습니다 😭

Reply:
제가 돌려보면 `org.springframework.web.util.NestedServletException: Request processing failed; nested exception is org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.binding.BindingException: Parameter 'isRocket' not found. Available parameters are [arg2, arg1, arg0, param3, param1, param2]` 이런 오류가 나더라구요~ 제 생각에는 저번처럼 ObjectMapper을 통한 객체 변환시에 발생한 것 같습니다!

Reply:
```suggestion
        List<Product> getProductsByIsRocket(@Param("isRocket") boolean isRocket,@Param("pageStart") int pageStart,@Param("listSize") int listSize);
```
이런 식으로 수정하였더니 파라미터를 잘 찾아가 테스트가 성공했습니다! 참고해주세요😃

Reply:
저는 저 어노테이션 없이도 테스트 성공해요!
시은님 컴퓨터에서는 mybatis 가 파라미터를 못찾나보네요 .... ㅠㅠ
일단 시은님을 위해 어노테이션은 추가해두겠습니다 😀

Reply:
제가 명령 실행자를 `IntelliJ IDEA`로 설정해두어 그랬었네요😭 `gradle`로 실행하니 에러없이 잘 돌아갑니다! 번거롭게 해드려 죄송합니다

Reply:
해결되어 다행입니다 😀